### PR TITLE
add a wait to make sure ssh is ready

### LINKED ISSUE #806: Tests should retry ssh to workers
Conformance tests on GCE don't retry reaching a worker node. It fails the job.

```
[621017.355592] golang[6]: Waiting 30s before retrieving worker metadata
[621047.356859] golang[6]: Getting worker public IP
[621179.871318] golang[6]: ssh: connect to host 35.193.103.109 port 22: Connection timed out
[621179.872332] golang[6]: lost connection
```

https://jenkins-kube-lifecycle.prod.coreos.systems/job/bootkube-dev/55/console

### PR DESCRIPTION
**Pull request by rphillips**
Fixes #806

### CODE REVIEW

**Comment by rphillips** [comment]
coreosbot run e2e

**Comment by rphillips** [comment]
ready for review

**Comment by colemickens** [comment]
`init-master.sh` does scp/ssh too. Should it also wait?

**Comment by colemickens** [comment]
nit: maybe pull it into lib.sh and then source it into both `init-*.sh` for de-dupe? might be overkill.

lgtm either way.

**Comment by rphillips** [comment]
good suggestion... these scripts are meant to be run standalone on different machines, so for simplicity I think the duplication is bearable.

**Comment by rphillips** [comment]
coreosbot run e2e

File: hack/quickstart/init-master.sh [hunk 1]
```diff
@@ -16,6 +16,27 @@ function usage() {
     exit 1
 }
 
+function retry_cmd() {
```
Comment (xiang90) [inline hack/quickstart/init-master.sh]: not a fan of duplicated code. but probably not your issue or specific to this commit. i know the old script has quite some duplication already.

**Pull request merged**